doc/cookbook/db-config.rst

```diff
@@ -149,4 +149,11 @@ Configuring the client in ``pipeline.yaml``
 -------------------------------------------
 
 Check out the :doc:`../get-started/sql-pipeline` to learn how to configure
-the database client in your ``pipeline.yaml`` file.
\ No newline at end of file
+the database client in your ``pipeline.yaml`` file. 
+
+SQL Examples
+---------------
+To see some examples using SQL connections, see this:
+1. A complete `example: <https://github.com/ploomber/projects/tree/master/cookbook/sql-dump>`_.
+2. `https://github.com/ploomber/projects/tree/master/templates/etl`_.
+3. `https://github.com/ploomber/projects/tree/master/templates/spec-api-sql`_.
```

Comment:
awesome! this looks better. can you update the text descriptions:

1. A short example that dumps data
2. A complete ETL pipeline
3. A SQL pipeline

Reply:
yes sure!

Reply:
done, thanks!